### PR TITLE
[release-2019.6.x] add node affinity to the UI

### PR DESCRIPTION
**Pull request by yashbhutwala**
This is a manual cherry-pick of #665 

/assign @rrati

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/681#" title="Author self-approved">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.6.x/OWNERS)~~ [yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: cmd/operator-ui/go.mod [hunk 1]
```diff
@@ -40,17 +39,18 @@ require (
 	github.com/gobuffalo/mw-i18n v0.0.0-20190224203426-337de00e4c33
 	github.com/gobuffalo/mw-paramlogger v0.0.0-20190224201358-0d45762ab655
 	github.com/gobuffalo/packr/v2 v2.5.1
+	github.com/gobuffalo/suite v2.8.1+incompatible
```
Comment (rrati) [inline cmd/operator-ui/go.mod]: Please add the new deps to the replace section

Comment (yashbhutwala) [reply cmd/operator-ui/go.mod]: This is automatically added via `make buffalo`.  Why are putting the same dep info in both `replace` and `require` block?

Comment (yashbhutwala) [reply cmd/operator-ui/go.mod]: Done, but I think we should redo `go.mod` files in both main and operator-ui and keep them consistent across master and release branches, probably from 2019.8.0 onwards.

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/681#pullrequestreview-263282800" title="LGTM">rrati</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/681#" title="Author self-approved">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.6.x/OWNERS)~~ [rrati,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**